code/style.css

```diff
@@ -1,4 +1,385 @@
 html {
-  background: #0025ff;
+  background: white;
   color: #fff;
+  font-family: 'Roboto Mono', monospace;
 }
+
+.wrapper{
+  justify-content: center;
+  display: flex;
+flex-direction: column;
+animation: fadeIn 5s;
+opacity: 1;
+}
+@keyframes fadeIn{
+  from{
+    opacity: 0;
+  }
+  to{
+    opacity: 1;
+  }
+  }
+
+header{
+display: flex;
+justify-content: space-evenly;
+align-self: center;
+align-items: center;
+flex-direction: row;
+}
+
+.meny{
+  display: inline;
+}
+.menu-line{
+  width: 35px;
+  height: 5px;
+  background-color: #25838A;
+  margin: 6px 0;
+  border-radius: 10px;
+  align-self: flex-end;
```

Comment:
as in line 26, is this needed? nothing changes when I remove it in the dev tool. But there might be some smart function that I've missed :)